internal/render/renderer.go

```diff
@@ -163,14 +163,14 @@ func (r *CliRenderer) RenderConfiguration(scope string, config map[string]string
 	r.tableWriter.Render()
 }
 
-func (r *CliRenderer) RenderRegisteredProcesses(registeredProcesses []processregistry.RegisteredProcess) {
+func (r *CliRenderer) RenderRegisteredProcesses(registeredProcesses []*processregistry.RegisteredProcess) {
 	if len(registeredProcesses) < 1 {
 		r.logger.Info("No processes found.")
 		return
 	}
 
 	r.tableWriter.SetHeader([]string{
-		"Process Name", "Process Version", "Process type", "Image", "Upload Date", "Owner",
+		"Name", "Version", "Type", "Image", "Upload Date", "Owner", "Status",
```

Comment:
I'd move the STATUS column just before the VERSION.

```suggestion
		"Name", "Version", "Status", "Type", "Image", "Upload Date", "Owner",
```